controllers/upgrade_controller.go

```diff
@@ -223,7 +223,9 @@ func (r *UpgradeReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	}))
 
 	// Watch for spec and annotation changes
-	nodePredicates := builder.WithPredicates(predicate.AnnotationChangedPredicate{})
+	nodePredicates := builder.WithPredicates(
```

Comment:
for node predicates, we care about node labels only no ?
when do we want to enqueue based on annotation change ?

the upgrade state is stored in label today.

Reply:
from controller side we care about nvidia.com/mofed-driver-upgrade-sate label only no ?